contracts/contracts/RequestManager.sol

```diff
@@ -738,6 +744,43 @@ contract RequestManager is Ownable, LpWhitelist, CrossDomainRestrictedCalls {
         request.filler = filler;
         request.fillId = fillId;
 
+        request.invalidFillIds[fillId] = false;
+
         emit RequestResolved(requestId, filler, fillId);
     }
+
+    /// Mark the fill identified by ``requestId`` and ``fillId`` as invalid.
+    ///
+    /// .. note::
+    ///
+    ///     This function is callable only by the native L2 messenger contract,
+    ///     which simply delivers the message sent from L1 by the
+    ///     Beamer's L2 :sol:interface:`messenger <IMessenger>` contract.
+    ///
```

Comment:
I would remove this note since it is no longer correct.

Reply:
why is this no longer correct?

Reply:
Because it's not the native L2 messenger contract that is going to be calling this in general (e.g. Arbitrum).